### DISCUSSION TITLE
How to specify the port in MkDocs

### DISCUSSION THREAD

**Discussion by Njoxpy** [mkdocs/mkdocs#3706] [Q&A] [answered]
Hello

I'm trying to customize the port for my MkDocs server but couldn't find a clear explanation in the documentation. Could someone please provide guidance on how to specify the port when running MkDocs?

**Comment by pawamoy** [comment, marked as answer]
You're looking at the `-a` option, see https://www.mkdocs.org/user-guide/cli/#mkdocs-serve:

> -a, --dev-addr 	text 	IP address and port to serve documentation locally (default: localhost:8000)

Example:

```bash
mkdocs serve -a localhost:9999
```
